components/button/index.js

```diff
@@ -26,11 +26,24 @@ class Button extends Component {
 	}
 
 	render() {
-		const { href, target, isPrimary, isLarge, isToggled, className, disabled, ...additionalProps } = this.props;
+		const {
+			href,
+			target,
+			isPrimary,
+			isSecondary,
+			isLarge,
+			isSmall,
+			isToggled,
+			className,
+			disabled,
+			...additionalProps, // eslint-disable-line comma-dangle
```

Comment:
I guess that sort of makes sense, though it would be nice to have a better error message.  I'll incorporate this change into #2205.